test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -194,6 +194,12 @@ def test_framework_version_cpu(image):
         else:
             if "neuron" in image:
                 assert tag_framework_version in output.stdout.strip()
+            if all(_string in image for _string in ["pytorch", "habana", "synapseai1.3.0"]):
+                # Habana Pytorch version looks like 1.10.0a0+gitb488e78 for SynapseAI1.3 PT1.10.1 images
```

Comment:
is this intentional?

Reply:
Yes, I think for PT1.10.1 images, the PT version seen within the image is `1.10.0a0+gitb488e78`. I think this is from the PT side (https://github.com/pytorch/pytorch/blob/v1.10.1/version.txt)